sources/engine/Stride.VirtualReality/OpenXR/OpenXRExt_FB_passthrough.cs

```diff
@@ -37,7 +36,7 @@ public OpenXRExt_FB_Passthrough(XR xr, Session session, Instance instance)
 
             bool TryGetProcAddress(string n, out nint fptr)
             {
-                PfnVoidFunction function;
+                PfnVoidFunction function = default;
```

Comment:
This appears to be unrelated to this specific PR concerning the GeometricPrimitive.Torus.New method.

It might be more appropriate to create a separate PR if you wish to suggest a change in OpenXR.

Reply:
I giveUp it